### ISSUE TITLE
when training 3D resnet18, controlling random seed is not availabel

### ISSUE THREAD

**Issue by chenxiahan** [pytorch/pytorch#61701]
When I trained 3D resnet, two repeated experiments had different results. And the deeper the network, the greater the error.

The code of controlling random seed is :
```
    random.seed(seed)
    np.random.seed(seed)
    os.environ['PYTHONHASHSEED'] = str(seed)
    torch.manual_seed(seed)
    torch.cuda.manual_seed(seed)
    torch.cuda.manual_seed_all(seed)
    torch.backends.cudnn.deterministic = True
    torch.backends.cudnn.benchmark = False
```

I have conducted many experiments:
1. changing 3D resnet to 2D  --> no error
2. using easier 3D model --> smaller error
3. using deeper 3D model --> greater error

pytorch version 1.6.0 and 1.1.0 have the same issue.

**Comment by gchanan** [comment]
Please see https://pytorch.org/docs/stable/generated/torch.use_deterministic_algorithms.html.
